macros/anaUltraThreshold.py

```diff
@@ -18,7 +20,10 @@
                   help="Make plots vs channels instead of strips", metavar="channels")
 parser.add_option("-p","--panasonic", action="store_true", dest="PanPin",
                   help="Make plots vs Panasonic pins instead of strips", metavar="PanPin")
-
+parser.add_option("--chConfigKnown", action="store_true", dest="chConfigKnown",
```

Comment:
Tried already, implementation was not so trivial.  Containers used in ultraThreshold.py are indexed only by vfat position.  While a --perchannel option exists larger updates to the code would be needed for this solution.

Proposed solution I think is easier and requires less modification.  Additionally anaUltraThreshold.py works with or without this additional input due to these option flags.